src/datepicker/Datepicker.vue

```diff
@@ -11,6 +11,7 @@
       @focus="renderView(startingView)"
       @click="renderView(startingView)"
     />
+    <i v-show="clearable" class="clearable" @click="clearInput()">&times;</i>
```

Comment:
That is a good default, but I think a lot of users would want to provide something themselves. I think a `<slot>` with this as a default content would be a better fit.